homeassistant/components/hue_ble/strings.json

```
@@ -0,0 +1,26 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Enter a name for the device and the MAC address of the light. Make sure the light is in Alexa/Google pairing mode.",
```

You could provide a link for further information on how to enable the pairing mode, either to a Phillips Hue page or a link with anchor to the documentation of the integration (provide the url as placeholder, links should not go into the translation itself)
```suggestion
        "description": "Enter a name for the device and the MAC address of the light. Make sure the light is in [Alexa/Google pairing mode.]({url_pairing_mode})",
```